diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/SingleLineStatementsRule.kt

```diff
@@ -72,26 +75,19 @@ class SingleLineStatementsRule : Rule("statement") {
 
     private fun findWrongText(node: ASTNode): String {
         var text: MutableList<String> = mutableListOf()
-        var prevNode: ASTNode? = node.treePrev
-        while (prevNode != null) {
-            val listText = prevNode.text.split("\n")
-            text.add(listText.last())
-            if (listText.size > minListTextSize)
-                break
-            prevNode = prevNode.treePrev
-        }
+        val nextNode: ASTNode? = node.parent({ it.treeNext != null }, false) ?: node
+        (node.siblings(false).map { sequenceNode -> sequenceNode.text.split("\n") }.takeWhileInclusive { listText ->
+            listText.size <= minListTextSize
+        }.forEach {
+            text.add(it.last())
+        })
```

Comment:
```suggestion
        node.siblings(false)
            .map { it.text.split("\n") }
            .takeWhileInclusive { it.size <= minListTextSize }
            .forEach {
                text.add(it.last())
            }
```
Placing these long chained calls on separate lines make whole statement more readable. Alsom, I don't see why named arguments are needed here. And why parentheses around whole construction?